config/nginx-config/default-pages/50x.html

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html>
+<head>
+<title>50x Error</title>
+<style>
+    body {
+        width: 80%;
+        margin: 0 auto;
+        font-family: Tahoma, Verdana, Arial, sans-serif;
+        background-color: #2b303b;
+        color: #c0c5ce;
+    }
+    pre, code {
+        background: #2b303b;
+        border-radius: 4px;
+        border: 1px solid #4f5b66;
+    }
+    code {
+        color: #a3be8c;
+        padding: .1em .2em;
+    }
+    .box {
+        border-radius: 4px;
+        padding: 1.5em;
+        background: #343d46;
+        box-sizing: border-box;
+        margin-bottom: 1em;
+        margin-top: 1em;
+    }
+    h1 {
+        color:white;
+    }
+</style>
+</head>
+<body>
+  <div class="box">
+    <h1>An error occurred in VVV.</h1>
+    <p>This could be because of:
+    <ul>
+      <li>The <code>vagrant provision</code> didn't finished yet.</li>
+      <li>The last provision failed, you can find the logs in <code>VVV/log/provisioners</code>.</li>
+      <li>You didn't update rightly VVV, use <code>git pull</code> and try a new <code>vagrant provision</code>.</li>
+    </ul></p>
+    <p><em>Faithfully yours, VVV team.</em></p>
+  </div>
```

Comment:
```suggestion
    <p>This could be because:</p>
    <ul>
      <li>Provisioning has not finished yet.</li>
      <li>The last provision failed, you can find the logs in <code>log/provisioners</code>.</li>
      <li>You made modifications to a sites Nginx config that contain mistakes.</li>
    </ul>
    <p>Double check the address, fix any Nginx configs you might have modified, and if you still have problems get in touch on GitHub or Slack</p>
  </div>
```